### PR TITLE
Remove elastic search indexer

### PR DESCRIPTION
**Pull request by jorgegorka**

### CODE REVIEW

File: Cheffile [hunk 1]
```diff
@@ -14,4 +14,3 @@ cookbook 'rbenv', git: 'https://github.com/aminin/chef-rbenv'
 cookbook 'phantomjs'
 
 cookbook 'java'
```
Comment (jackregnart) [inline Cheffile]: Without ElasticSearch. Do we still need java?

Comment (jorgegorka) [inline Cheffile]: I hope we don't :-)

Comment (tripledub) [inline Cheffile]: probably not, we probably don't need chef/vagrant not that Shaun has a proper dev environment

Comment (tripledub) [inline Cheffile]: (The `cheffile` is for local dev with `vagrant`)

Comment (jackregnart) [inline Cheffile]: Ah, well nevermind then.

**Pull request merged**